arangodb/content.md

```diff
@@ -0,0 +1,136 @@
+# What is ArangoDB?
+ArangoDB is a multi-model, open-source database with flexible data models for documents, graphs, and key-values. Build high performance applications using a convenient SQL-like query language or JavaScript extensions. Use ACID transactions if you require them. Scale horizontally and vertically with a few mouse clicks.
+
+The supported data models can be mixed in queries and allow ArangoDB to be the aggregation point for the data request you have in mind.
+
+> [arangodb.com](https://arangodb.com)
+
+%%LOGO%%
+
+Key Features in ArangoDB
+------------------------
+
+**Multi-Model**
+Documents, graphs and key-value pairs — model your data as you see fit for your application.
+
+**Joins**
+Conveniently join what belongs together for flexible ad-hoc querying, less data redundancy.
+
+**Transactions**
+Easy application development keeping your data consistent and safe. No hassle in your client.
+
+Joins and Transactions are key features for flexible, secure data designs, widely used in RDBMSs that you won’t want to miss in NoSQL products. You decide how and when to use Joins and strong consistency guarantees, keeping all the power for scaling and performance as choice. 
+
+Furthermore, ArangoDB offers a microservice framework called [Foxx](https://www.arangodb.com/foxx) to build your own Rest API with a few lines of code.
+
+ArangoDB Documentation
+- [ArangoDB Documentation](https://www.arangodb.com/documentation)
+- [ArangoDB Tutorials](https://www.arangodb.com/tutorials)
+
+## How to use this image
+
+### Start an `ArangoDB` instance
+
+In order to start an ArangoDB instance run
+
+	unix> docker run -d --name arangodb-instance -d arangodb/arangodb
```

Comment:
There is an extra `-d` here.  Swap to just arangodb instead of arangodb/arangodb (a few places throughout the file).